docs/aws/README.md

```diff
@@ -132,29 +144,31 @@ spec:
             requests:
               cpu: 1
 EOF
-kubectl scale deployment inflate --replicas 5
-kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
+## Automatic Node Provisioning 
 
-You can see what EC2 instance type was added to your cluster from Karpenter with
-```bash
-kubectl get no -L "node.kubernetes.io/instance-type"
-```
+Scale the previous deployment to 5 replicas, and begin watching the karpenter pod for log events. 
 
-If you scale down the deployment replicas the instance will be terminated after 30 seconds (ttlSeconds).
 ```bash
-kubectl scale deployment inflate --replicas 0
+kubectl scale deployment inflate --replicas 5
+kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
 
-Or you can manually delete the node with
+## Automatic Node Deprovisioning
+
+Now, delete the deployment. After 30 seconds (`ttlSecondsAfterEmpty`), karpenter should deprovision the now empty nodes. 
 
-> Karpenter automatically adds a node finalizer to properly cordon and drain nodes before they are terminated.
 ```bash
-kubectl delete node $NODE_NAME
+kubectl delete deployment inflate
+kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
 
-### Cleanup
-> To avoid additional costs make sure you delete all ec2 instances before deleting the other cluster resources.
+# Cleanup
+
+It's important to both delete the cluster instances and the cluster control plane. AWS charges for both. 
+
+Delete cluster workloads and instances: 
```

Comment:
This doesn't delete instances. This just uninstalls karpenter and deletes the IAM resources deployed by CFN.

Reply:
I currently think deleting the cluster deletes associated managed node groups. Does Karpenter modify this behavior? I hadn't considered that impact.

Reply:
Yeah this is actually quite interesting. Since we're not using node groups, these instances may leak.

Reply:
I'll use that `delete nodes` cmd as an example, and call this out.